base/utils/R/sensitivity.R

```diff
@@ -296,6 +296,17 @@ write.sa.configs <- function(defaults, quantile.samples, settings, model,
               file = file.path(settings$rundir, run.id, "README.txt"), 
               sep = "")
           
+          # I check to make sure the path under the met is a list. if it's specified what met needs to be used in 'met.id' under sensitivity analysis of pecan xml we used that otherwise, I use the first met.
+          if (is.list(settings$run$inputs$met$path)){
+            # This checks for met.id tag in the settings under sensitivity analysis - if it's not there it creates it. Then it's gonna use what it created.
+            if (is.null(settings$sensitivity.analysis$met.id))  settings$sensitivity.analysis$met.id <- 1
+             
+              settings$run$inputs$met$path <- settings$run$inputs$met$path[settings$sensitivity.analysis$met.id]
```

Comment:
I guess then makes sure the result is just one string with the final path, so if if there is say a path$path1 this will return path1 as path as a single string so I think this should be good, if its a string

Reply:
It's gonna be a string for sure, if there is a bunch of paths under met$path like `<path1>, <path2> ` then either it uses the `met.id` tag in the settings or it just takes the first one and puts one path string in the met$path.